src/colors/stories/colors.stories.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import isString from 'lodash/isString';
+import isEmpty from 'lodash/isEmpty';
+import isPlainObject from 'lodash/isPlainObject';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+
+import * as Colors from '../src/Colors';
+import { Card, CardSegment } from '../../layout';
+
+const ColorGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+`;
+
+const ColorBlock = styled.div`
+  background-color: ${(props) => props.color};
+  color: ${(props) => props.textColor};
+  cursor: pointer;
+  height: 100px;
+  padding: 10px;
+  width: 100%;
+`;
+
+export default {
+  title: 'Colors',
+};
+
+export const LUKColors = () => {
+  const getTextColor = (hexcolor :string) :string => {
+    const r = parseInt(hexcolor.substr(1, 2), 16);
+    const g = parseInt(hexcolor.substr(3, 2), 16);
+    const b = parseInt(hexcolor.substr(5, 2), 16);
+    const yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000;
+    return (yiq >= 128) ? 'black' : 'white';
+  };
```

Comment:
This is pretty cool. Did you consider using the utility from `polished` (no idea if it works well)?

https://polished.js.org/docs/#readablecolor

Reply:
it works! I'll use it!